Adyen/Model/Checkout/CheckoutPaymentsAction.cs

```diff
@@ -451,6 +460,11 @@ public bool Equals(CheckoutPaymentsAction input)
                     (this.ShopperEmail != null &&
                     this.ShopperEmail.Equals(input.ShopperEmail))
                 ) &&
+                (
+                    this.SdkData == input.SdkData ||
+                    (this.SdkData != null &&
+                    this.SdkData.Equals(input.SdkData))
+                ) &&
```

Comment:
can you please add sdkData to the GetHashCode method?

Reply:
Done